tasks/data_generation_base.py

```diff
@@ -0,0 +1,104 @@
+import os
```

Comment:
This module can be replaced by cumulusci.tasks.bulkdata.base_generate_data_task, yeah?

Reply:
Deleted now.